src/app/api/user/page_views/route.ts

```diff
@@ -0,0 +1,45 @@
+import prisma from "@/lib/prisma";
+import { NextResponse } from "next/server";
+
+export async function POST(req: Request) {
+  const { searchParams } = new URL(req.url);
+  const storyId = searchParams.get("story_id");
```

Comment:
For POST requests, we prefer using body parameters rather than path parameters.

Reply:
Fixed the issue